website/docs/language/expressions/types.mdx

```diff
@@ -21,13 +21,16 @@ The Terraform language uses the following types for its values:
   numbers like `15` and fractional values like `6.283185`.
 * `bool`: a boolean value, either `true` or `false`. `bool` values can be used in conditional
   logic.
-* `list` (or `tuple`): a sequence of values, like
-  `["us-west-1a", "us-west-1c"]`. Elements in a list or tuple are identified by
-  consecutive whole numbers, starting with zero.
-* `map` (or `object`): a group of values identified by named labels, like
+* `list`: a sequence of values, like `["us-west-1a", "us-west-1c"]`. Identify elements in a list with consecutive whole numbers, starting with zero.
+* `set`: a collection of unique values that do not have any secondary identifiers or ordering.
+* `map`: a group of values identified by named labels, like
```

Comment:
`object` was removed from the definition here, but the example is an `object` and not a `map`